pkg/controller/baremetalhost/baremetalhost_controller.go

```diff
@@ -43,11 +45,22 @@ const (
 
 var runInTestMode bool
 var runInDemoMode bool
+var maxConcurrentReconciles int = 1
 
 func init() {
 	flag.BoolVar(&runInTestMode, "test-mode", false, "disable ironic communication")
 	flag.BoolVar(&runInDemoMode, "demo-mode", false,
 		"use the demo provisioner to set host states")
+
+	var err error
+	maxConcurrentReconciles, err = strconv.Atoi(os.Getenv("OPERATOR_CONCURRENCY"))
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "OPERATOR_CONCURRENCY variable not found. Using default: 1")
+	}
+	if maxConcurrentReconciles <= 0 {
```

Comment:
This could be combined with the conditional above so we only need to set the default in one place?

Reply:
Yes, this makes sense. Thanks.